valhalla/baldr/nodeinfo.h

```diff
@@ -206,7 +206,7 @@ class NodeInfo {
   bool can_contract() const {
     return edge_count() >= 2 && intersection() != IntersectionType::kFork &&
            type() != NodeType::kGate && type() != NodeType::kTollBooth &&
-           type() != NodeType::kTollGantry && type() == NodeType::kSumpBuster;
```

Comment:
i was sure it was going to be here but after looking at it 4 times i still didnt see it :smile:

Reply:
Not sure why the OSX build is still failing though. I'm developing on Mac myself so that is where I wrote the test in the first place 🤔